File: 4) C#/pagamento.cs
using System;

namespace Pagamento
{
	class Program
	{
		static void Main( string[ ] args ) 
		{
			int horas, pagamento, salario;
			
			Console.WriteLine( "Digite suas horas trabalhadas: " );
			horas = Convert.ToInt32( Console.ReadLine( ) );
			Console.WriteLine( "Digite seu salario por hora de trabalho: " );
			salario = Convert.ToInt32( Console.ReadLine( ) );
			
			pagamento = salario * horas;
			Console.WriteLine( $"Pagamento: {pagamento}" );
		}
	}
}

File: 6) C E C++/pagamento.cpp
#include <iostream>

int main( ) {
	int pagamento, salario, horas;
	
	std::cout << "Digite o seu salario por hora trabalhada: ";
	std::cin >> salario;
	std::cout << "Digite suas horas trabalhadas: ";
	std::cin >> horas;
	
	pagamento = horas * pagamento;
	std::cout >> "Pagamento de: " << pagamento << std::endl;
}